sonar-core/src/main/java/org/sonar/core/user/DefaultUser.java

```diff
@@ -27,8 +27,10 @@
 import org.sonar.api.user.User;
 
 /**
- * @since 3.6
+ * Used only by unit tests. To be replaced by UserDto.
+ * @deprecated
  */
+@Deprecated
 public class DefaultUser implements User {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)